proofreading/make_bigcat_infected_cells_project.py

```diff
@@ -23,8 +23,9 @@ def fragment_segment_lut(segmentation, infected_labels):
 
 def to_uint8(raw):
     raw = raw.astype('float32')
-    raw -= raw.min()
-    raw /= raw.max()
+    raw_min = raw.min()
+    raw -= raw_min
+    raw /= raw.max() - raw_min
```

Comment:
I don't quite understand this change, the initial normalization should be fine, no?

Reply:
Sorry I forgot to comment this change. I just used the standard 0-1 normalization formula. With the other one you don't subtract by the minimum in line 28, so you get a range between 0 and something less than one. In case an image has a constant offset (might never be the case here) this should prevent the dynamic range shrinkage.

Reply:
Never mind, I was updating the branch and I realized I got a blunder on the normalization.